spec/views/sufia/admin/admin_sets/_form_metadata.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.describe 'sufia/admin/admin_sets/_form_metadata.html.erb', type: :view do
+  let(:admin_set) { create(:admin_set) }
+  let(:permission_template) { Sufia::PermissionTemplate.find_or_create_by(admin_set_id: admin_set.id) }
+  before do
+    @form = Sufia::Forms::AdminSetForm.new(admin_set, permission_template)
+    render 'form'
```

Comment:
The simple form is setup in _form using
```
    <%= simple_form_for @form, url: [sufia, :admin, @form] do |f| %>
```
and is passed to _form_metadata using
```
   <%= render 'form_metadata', f: f %>
```

I tried doing the setup in the test using
```
  before do
    @form = Sufia::Forms::AdminSetForm.new(admin_set, permission_template)
    simple_form_for @form, url: [sufia, :admin, @form] do |f|
      render 'form_metadata', f: @form
    end
  end
```
But this fails with error: undefined method `simple_form_for'

Do you have a suggestion of how to set up the test to call `render 'form_metadata'` directly instead of through `render 'form'`?